static/js/play.js

```diff
@@ -311,6 +311,12 @@ function checkForFind(e) {
     }
 }
 
+function disableFind(e) {
+    console.log(e);
+    if ([114, 191, 222].includes(e.keyCode) || ((e.ctrlKey || e.metaKey) && e.keyCode == 70)) { 
+        e.preventDefault();
```

Comment:
```
        if (ctrlfwarnings == true) {
            app.$data.finished = true;
            app.$data.caught = true;
        }
```
Need to add this part in from checkForFind if we want to end people's runs here. Although we can make an argument for just disabling it without ending the run as well.

Reply:
Personally, I think we should go the middle path, and just like flash a warning or something rather than end the run. If we want to scare people more, we can say something like 'this will be recorded'